onnxruntime/contrib_ops/cpu/qlinear_pool.cc

```diff
@@ -485,6 +490,24 @@ struct QLinearPoolNhwc3DTask final {
   }
 };
 
+template <typename T8Bits>
+void dequantize_array(int64_t N, const T8Bits* input, float scale, T8Bits zero_point, float* output, ThreadPool* tp) {
+  if (N > 512) {
+    float dequantize_lookup[256];
+    for (int i = 0; i < 256; ++i) {
```

Comment:
looks like we are building another lookup table, is there scenario where the scale and zero point are fixed that we can cache and reuse the table, like in
QlinearBuildLookupTable?

Reply:
Table is different, it is float 32. I feel the logic is two small to generalize it now. And the lookup logic is generalized there.
Maybe it is better to do it when we want to finalize the dequantize() operater if needed.